libservice/headers/service/Service.h

```diff
@@ -26,10 +26,12 @@ struct Service {
 	std::string endpoint;
 	uint32_t internalClientsNumber{0u};
 	uint32_t externalClientsNumber{0u};
+	std::string domain;
```

Comment:
I would rather keep counts (internal/external clients number) as the last elements in the structure.

Reply:
Could you explain why? This order is used as the JSON output order (which was accepted by you earlier) and I wanted the orders to coincide with each other:
`{"service":[{"pid":134011,"endpoint":"google.com:4443/test/other","externalClientsNumber":1,"domain":"google.com","scheme":"https"}]}`

Reply:
Yes, in my opinion this order also would be better with number values in the end. For me it's just more natural/common formatting - parameters (less changing) first, then values. 

I'm sorry for not catching it earlier - I just focused strictly on scheme/domain values.
If the effort to change it is not worth it in your opinion, let's keep it this way.